app/views/tags/index.html.erb

```diff
@@ -1,28 +1,28 @@
-<% title "Tags on #{community_name}" %>
+<% title t("core.tag").pluralize %>
 
 <%= content_for :page_meta do %>
   <link rel="canonical" href="<%= app_url("/tags") %>" />
-  <meta name="description" content="Tags to follow on <%= community_name %>">
+  <meta name="description" content="<%= t("core.tags_to_follow", community_name: community_name) %>">
   <%= meta_keywords_default %>
 
   <meta property="og:type" content="article" />
   <meta property="og:url" content="<%= app_url("/tags") %>" />
-  <meta property="og:title" content="Tags to follow on <%= community_name %>" />
+  <meta property="og:title" content="<%= t("core.tags_to_follow", community_name: community_name) %>" />
   <meta property="og:image" content="<%= Settings::General.main_social_image %>" />
-  <meta property="og:description" content="<%= community_name %> is great!" />
+  <meta property="og:description" content="<%= t("core.community_name_is_great", community_name: community_name) %>" />
```

Comment:
This basically strikes me as placeholder content we never fixed. I don't know if we should sit around and litigate all the copy right in these PRs, but if we merge this, I'll probably think about this and try and follow up with some new ideas for this area. 😄